src/numeric.rb

```diff
@@ -184,4 +184,7 @@ def numerator
   def denominator
     to_r.denominator
   end
+
+  undef :positive?
```

Comment:
Yeah think thats right, `positive?` & `negative?` are [removed from Complex](https://github.com/ruby/ruby/blob/5ccb625fbbd1e774636a9fdbe0bf1c3d38e085d5/complex.c#L2400) in Ruby, but not from `Numeric`, so need to move these to `Complex`

Reply:
oh, lol, oops. was editing the wrong file